content/chat/getting-started/javascript.textile

```diff
@@ -0,0 +1,195 @@
+---
+title: "Getting started: Chat and JavaScript"
```

Comment:
Something thats crossed my mind - do we want to call out "TypeScript" here? Yes the SDK is technically a JS SDK, but TypeScript is more or less the de-facto nowadays.

Reply:
Just in the title, or in the nav as well? For the title we could do `JavaScript/TypeScript`, but that might look a bit odd in the nav.